File: Unit-2/week4/day5/Retrofit-11/app/src/main/java/com/example/retrofit11/PostViewHolder.java
package com.example.retrofit11;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

public class PostViewHolder extends RecyclerView.ViewHolder {

    private TextView mtvName;
    private TextView mtvEmail;
    private TextView mtvBody;

    public PostViewHolder(@NonNull View itemView) {
        super(itemView);
        initViews(itemView);

    }

    private void initViews(View itemView) {
        mtvBody = itemView.findViewById(R.id.tvBody);
        mtvEmail = itemView.findViewById(R.id.tvEmail);
        mtvName = itemView.findViewById(R.id.tvEmail);

    }

    public void setData(ResponseModel responseModel){
        mtvEmail.setText(responseModel.getEmail());
        mtvName.setText(responseModel.getName());
        mtvBody.setText(responseModel.getBody());
    }
}

File: Unit-3/week4/Day2/KothinAndroid/app/src/main/java/com/example/kotlinandroid/PostViewholder.kt
package com.example.kotlinandroid

import android.view.View
import androidx.recyclerview.widget.RecyclerView

class PostViewholder(private val view : View) : RecyclerView.ViewHolder(view) {

    fun setData(responseDTOItem: ResponseDTOItem){
       view.apply {
           tvName.text = responseDTOItem.name
           tvBody.text = responseDTOItem.body
           tvEmail.text = responseDTOItem.email
       }
    }
}